### DISCUSSION TITLE
can't get soulseek and nicotine+to work

### DISCUSSION THREAD

**Discussion by chuks1111** [nicotine-plus/nicotine-plus#1930] [Q&A]
how can i get soulseek to work, i am trying nicotine+ and it's not working also it said UPnP was not working on nicotine

**Comment by slook** [comment]
UPnP is not required for Nicotine+ to work. You should be able to connect after entering a username and password. To suppress the UPnP error message you may disable UPnP in Preferences > Network.

If UPnP doesn't work on your network, then to get a better connection to more users you can forward the IP port Nicotine+ in your router config.

**Reply by chuks1111** [reply]
how do you forward the IP port? i use at talk talk router. i can get log in but where do i start?

**Reply by slook** [reply]
You can use Nicotine+ without forwarding any port, it will work okay this way. You can test by trying 'Search File' and double-clicking a result to Download it.

Instructions about how to setup port forwarding on your router is beyond the scope of this forum. You may find details about how to do this at various websites including link http://www.pcwintech.com/port-forwarding-guides or on your router manufacturers website.

**Comment by chuks1111** [comment]
Can you pls explain how? I got a talk talk router with BT. London boy.  Sent from Mail for Windows From: slookSent: 27 February 2022 22:42To: nicotine-plus/nicotine-plusCc: chuks1111; AuthorSubject: Re: [nicotine-plus/nicotine-plus] can't get soulseek and nicotine+to work (Discussion #1930) UPnP is not required for Nicotine+ to work. You should be able to connect after entering a username and password.To get a better connection to more users you can forward the IP port Nicotine+ in your router config.—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>
